src/runtime/bootstrap-lazy.ts

```diff
@@ -137,6 +137,8 @@ export const bootstrapLazy = (lazyBundles: d.LazyBundlesRuntimeData, options: d.
         patchSlotAppendChild(HostElement.prototype);
       }
 
+      patchTextContent(HostElement.prototype);
```

Comment:
Per Nick: Consider adding a build flag to do some risk mitigation here/evaluate with the customer